universal-application-tool-0.0.1/app/models/Question.java

```diff
@@ -231,4 +235,36 @@ private void setFieldsFromQuestionDefinition(QuestionDefinition questionDefiniti
   public boolean removeVersion(Version draftVersion) {
     return this.versions.remove(draftVersion);
   }
+
+  public boolean addTag(QuestionTag tag) {
```

Comment:
suggestion (blocking): please document this public method

Reply:
Done